core/src/main/java/nu/yona/server/rest/GlobalExceptionMapping.java

```diff
@@ -47,6 +48,8 @@
 	@ResponseBody
 	public ErrorResponseDto handleOtherException(Exception exception, HttpServletRequest request)
 	{
+		Optional<String> opt = Optional.empty();
+		logger.debug(opt.get());
 		return logUnhandledExceptionAndCreateErrorDto("completed with unknown exception", exception, request);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "opt.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://build.dev.yona.nu:9000/coding_rules#rule_key=squid%3AS3655)

Reply:
👍